server/opts.go

```diff
@@ -105,8 +105,8 @@ type Options struct {
 	HTTPPort         int           `json:"http_port"`
 	HTTPSPort        int           `json:"https_port"`
 	AuthTimeout      float64       `json:"auth_timeout"`
-	MaxControlLine   int           `json:"max_control_line"`
-	MaxPayload       int           `json:"max_payload"`
+	MaxControlLine   int32         `json:"max_control_line"`
+	MaxPayload       int32         `json:"max_payload"`
```

Comment:
See comment about Varz..

Reply:
You mean if they are setting there own variables to this value? Compiler may warn. If setting to local variable compiler will do the right thing..

Reply:
true if they do `mp := varz.MaxPayload`, but if they had defined `var mp int` and then `mp = varz.MaxPayload`, then they will get a compilation error.

Reply:
Understood. At the varz level I am ok with reverting back to int..

Reply:
But for options it will be the same if trying to assign some value that use load from some places and store that in a `int`.

Reply:
And to be clear, it's fine to "break" things, but just wonder if it is warranted in this context.